src/components/carousel/bookContentCarousel.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component, Fragment } from "react";
```

Comment:
![image](https://user-images.githubusercontent.com/34189742/83161684-c06c7400-a0d6-11ea-9cf8-6882fd2840b1.png)
Hi! I think you might want to fix the styling on the books a bit? We can make another card for this if you like :)?

Reply:
Yeah, another card would be good.

Reply:
Yes I need to fix it. :)

Reply:
> ![image](https://user-images.githubusercontent.com/34189742/83161684-c06c7400-a0d6-11ea-9cf8-6882fd2840b1.png)
> Hi! I think you might want to fix the styling on the books a bit? We can make another card for this if you like :)?

I fixed it.